skaffold.yaml

```diff
@@ -0,0 +1,77 @@
+# Skaffold - https://skaffold.dev/
+# ------------------------------- #
+# Installation options & profiles #
+# ------------------------------- #
+# - Istio    - `skaffold run`
+# - Gateways - `skaffold run -p ingress,egress`
+# - Addons:
+#   - kiali  - `skaffold run -p kiali`
+# ------------------------------- #
+# Development mode - skaffold dev #
+# ------------------------------- #
+apiVersion: skaffold/v2beta22
+kind: Config
+metadata:
+  name: istio-base
+deploy:
+  helm:
+    releases:
+      - name: istio-base
+        chartPath: manifests/charts/base
+        namespace: istio-system
+        createNamespace: true
+---
+apiVersion: skaffold/v2beta22
+kind: Config
+metadata:
+  name: istiod
+requires:
+  - configs: [istio-base]
+deploy:
+  helm:
+    releases:
+      - name: istiod
+        chartPath: manifests/charts/istio-control/istio-discovery
```

Comment:
Installing from local source is going to instill bad practices IMO. We do NOT support just checking out `master` on git and installing, it will be configured with `latest` dev image build which is not suitable. only the release tarballs on github should be used (or really, the helm repo)

Reply:
Updated